about.html

```diff
@@ -12,53 +12,89 @@
 </head>
 
 <body>
+
     <div class="container about-page">
-     
-    About
 
+    <header class="tophead">
+      <a href="index.html">
+        <img src="img/lambda-black.png" alt="site-logo">
+      </a>
+    <nav>
+        <a href="index.html">Home</a>
+        <a href="#">About</a>
+        <a href="#">Products</a>
+        <a href="#">Blog</a>
+        <a href="#">Contact</a>
+    </nav>    
+</header>
+
+<img class="jumbo-about" src="img/jumbo-about.png">
+ 
+     <div class="abouts">
+    <h3>About</h3>
+<p>
     Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
+</p>
+</div>
 
 
+<div class=mid4container>
+<section class="middle4">
     <img src="img/about-plan.png" alt="strategy">
     
-    Strategy
-    
+    <h3>Strategy</h3>
+    <p>
     Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
+    </p>
+   <div>
+    <a>Learn More</a>
+   </div>
+</section>
 
-    Learn More
-
-
-
+<section class="middle4">
     <img src="img/about-working.png" alt="strategy">
     
-    How We Work
-    
+    <h3>How We Work</h3>
+    <p>
     Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
+    </p>
+    <div>
+    <a>Learn More</a>
+    </div>
+  </section>
 
-    Learn More
-
-
-
+    <section class="middle4">
     <img src="img/about-office.png" alt="strategy">
     
-    Places We Work
-
+    <h3>Places We Work</h3>
+    <p>
     Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
+    </p>
+    <div>
+    <a>Learn More</a>
+    </div>
+  </section>
 
-    Learn More
-
+    <section class="middle4">
     <img src="img/about-meeting.png" alt="strategy">
     
-    Collaboration
-    
+    <h3>Collaboration</h3>
+    <p>
     Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-    Let's Work Together
-
+    </p>
+    <div>
+    <a>Learn More</a>
+    </div>
+    </section>
+    </div>
+
+    <section class="bottomsec">
+    <h3>Let's Work Together</h3>
+    <p>
     Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-        
+    </p>
+    </section>
+
```

Comment:
I might recommend making your classes a little more semantic. You understand what bottomsec means, but a future developer might not. Also adding comments into the code will help you stay organized.